exercises/practice/hello-world/src/lib.rs

```diff
@@ -1,5 +1,3 @@
-// The &'static here means the return type has a static lifetime.
-// This is a Rust feature that you don't need to worry about now.
 pub fn hello() -> &'static str {
-    "Goodbye, World!"
+    "Goodbye, Mars!"
```

Comment:
We're standardizing this text. See https://github.com/exercism/v3-launch/issues/46